tests/storage/storage.go

```diff
@@ -1245,6 +1245,7 @@ var _ = SIGDescribe("Storage", func() {
 				_, err := virtClient.CdiClient().CdiV1beta1().DataVolumes(util.NamespaceTestDefault).Create(context.Background(), dv, metav1.CreateOptions{})
 				Expect(err).ToNot(HaveOccurred())
 				labelKey := "testshareablekey"
+				labelValue := "testshareablevalue"
```

Comment:
Why did we need this change in the e2e tests?

Reply:
> -@alaypatel07 hi i kind figure out the reason why "**pull-kubevirt-e2e-k8s-1.22-sig-storage**" and "**pull-kubevirt-e2e-k8s-1.23-sig-storage**" test failed, it looks like test template is working fine before i add label value "**testshareablevalue**" it`s eqaul to following
> 
> > spec:
> > affinity:
> > podAntiAffinity:
> > preferredDuringSchedulingIgnoredDuringExecution:
> > - weight: 1
> > podAffinityTerm:
> > labelSelector:
> > matchExpressions:
> > - key: testshareablekey
> > operator: In
> > values:
> > - ""
> > topologyKey: kubernetes.io/hostname
> 
> I`m able to make the vm in running stat, but after I put "**testshareablevalue**" to values, then vmi will pending on schedule, i think that's why these 2 e2e end up in timeout because vm never get scheduled. I think i should not add value to label(my bad)

@rmohr You are right, we don't need to put a value there same story as above.

Reply:
the slipped away in rebase, good catch, will update.

Reply:
updated, PTAL